packages/components/src/components/header/header.stories.ts

```diff
@@ -53,285 +66,20 @@ export const Default = {
   }
 };
 
+/**
+ * Use the `fixed` attribute to set the header to a fixed position.
+ */
+
 export const Fixed = {
-  parameters: {
-    controls: { exclude: ['fixed', '--sd-header-inner-width'] }
-  },
-  render: (args: any) => {
-    return html` ${generateTemplate({
-        constants: [
-          {
-            type: 'attribute',
-            name: 'fixed',
-            value: 'true'
-          }
-        ],
-        args
-      })}
+  name: 'Fixed',
+  render: () => html`
+    <div>
+      <sd-header fixed>
+        <div class="slot slot--border slot--text">Default slot</div>
+      </sd-header>
       <div class="bg-neutral-100 p-8" style="margin-top: var(--sd-header-calculated-height)">
         This element automatically sets its position depending on the height of the header.
```

Comment:
What does it mean?
Is it refering to the gray background below the header?

Pls replace gray bg with white bg